default.json

```diff
@@ -14,7 +14,7 @@
             "logging_enabled": true,
             "application_is_private_build": true,
             "development_api_enabled": true,
-            "mls_support_enabled": true,
+            "mls_support_enabled": false,
```

Comment:
Is this change intentional? 😅

Reply:
No, in federation squad we always forgot to change it back because without it we cannot use app 😅